timetracker/entries/migrations/0001_initial.py

```diff
@@ -1,25 +1,51 @@
 # -*- coding: utf-8 -*-
```

Comment:
I don't know what exact problem you had with the migrations, but you _normally_ should not modify migration files.